jetbrains-core/it/software/aws/toolkits/jetbrains/utils/TestUtils.kt

```diff
@@ -28,9 +29,10 @@ fun executeRunConfiguration(
     executorId: String = DefaultRunExecutor.EXECUTOR_ID
 ): Output {
     val executor = ExecutorRegistry.getInstance().getExecutorById(executorId)
+    assertThat(executor).isNotNull
```

Comment:
If you use `kotlin.test.assertNotNull` you won't need the `!!` on line 35 cos it'll smart-cast (it uses a [contract](https://pspdfkit.com/blog/2018/kotlin-contracts/) to hint to the compiler)

Reply:
I wanted to use the same assertion lib that we were using for everything but that looks better, changed